README.md

```diff
@@ -19,23 +19,25 @@ Once the cloud resources are provisioned, see the [viya4-deployment](https://git
 
 ## Prerequisites
 
-Operational knowledge of:
+### Operational knowledge of:
 
 - [Terraform](https://www.terraform.io/intro/index.html)
 - [Docker](https://www.docker.com/)
 - [Microsoft Azure Cloud](https://azure.microsoft.com/)
 - [Kubernetes](https://kubernetes.io/docs/concepts/)
  
-### Required
+
+### Technical Prerequisites:
+
+This project supports running **either** via terraform installed on your local machine **or** via a docker container. 
 
 - Access to an **Azure Subscription** and [**Identity**](./docs/user/TerraformAzureAuthentication.md) with '*Contributor*' role
-- Terraform or Docker
-  - #### Terraform
+  - #### Terraform (4 dependencies)
     - [Terraform](https://www.terraform.io/downloads.html) - v0.13.6
     - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) - v1.18.8
     - [jq](https://stedolan.github.io/jq/) - v1.6
     - [Azure CLI](https://docs.microsoft.com/en-us/cli/azure) - v2.19.1 (optional -useful as an alternative to the Azure Portal)
-  - #### Docker
+  - #### Docker (1 dependency)
```

Comment:
Remove the text: (1 dependency)

Reply:
@andybouts when will this be addressed?

Reply:
It has been removed, must have been an oversight.  That said, I think this would be more clear to the end users if it said "Terraform Dependencies" and "Docker Dependencies" or "Terraform Prerequisites" and "Docker Prerequisites".